### PR TITLE
Show explanation for CTRs of > 100%

### PR DESCRIPTION
**Pull request by kelvin-chappell**

### CODE REVIEW

File: app/views/Application/layout.scala.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
         <link rel="shortcut icon" type="image/png" href="@routes.Assets.versioned("images/icons/favicon.png")"/>
         <link rel="stylesheet" type="text/css" href="@routes.Assets.versioned("build/main.css")"/>
         <link rel="stylesheet" href="https://unpkg.com/purecss@@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (LATaylor-guardian) [inline app/views/Application/layout.scala.html]: Do we need bootstrap aswell as purecss?

File: public/components/CampaignAnalytics/Analytics/CampaignReferral.js [hunk 2]
```diff
@@ -4,6 +4,19 @@ import "react-infinity-menu/src/infinity-menu.css";
...
 
+  explainImpossibleCTR(ctr) {
+    if (ctr > 1) {
+      const explanation = "The behaviour of the back button in the Safari browser is anomalous;"+
+        " following a link from a front, then using the back button" +
+        " to go back to the front, then following the same link again" +
+        " is being counted as a single impression and two clicks rather than two impressions and two clicks." +
+        " This should have a negligible effect except where the number of impressions is very low," +
+        " as in this case.";
+      return <span className="glyphicon glyphicon-warning-sign" title={explanation}></span>
```
Comment (LATaylor-guardian) [inline public/components/CampaignAnalytics/Analytics/CampaignReferral.js]: I think there are already files in campaign central with icons.

Comment (kelvin-chappell) [reply public/components/CampaignAnalytics/Analytics/CampaignReferral.js]: Good point.  I'll pull out the icon.

File: public/components/CampaignAnalytics/Analytics/CampaignReferral.js [hunk 3]
```diff
@@ -4,6 +4,19 @@ import "react-infinity-menu/src/infinity-menu.css";
 
 export default class CampaignReferral extends React.Component {
 
+  static explainImpossibleCTR(ctr) {
+    if (ctr > 1) {
+      const explanation = "The behaviour of the back button in the Safari browser is anomalous;"+
```
Comment (LATaylor-guardian) [inline public/components/CampaignAnalytics/Analytics/CampaignReferral.js:9]: Nice explanation!

**Comment by LATaylor-guardian** [comment]
:shipit: 👍

**Pull request merged**